### PR TITLE
Adds the card game Une.

### PR DESCRIPTION
**Pull request by DrSnips**
Does what it says on the tin.
The name is Une to avoid issues with the name Uno. I do need to do more testing with this in a live server so Id like to test merge this tosee how things work,
:cl:
* rscadd: Added Une. A card game that's definitely not Uno. Buy a deck from the Merch Computer or print it using the Rapid-Service-Fabricator.

### CODE REVIEW

**Comment by DrSnips** [comment]
The card's use a white base and are colored via code to avoid any confusion of why they look invisible.

File: code/modules/games/cards/une/une_cards.dm [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+		cards += new/obj/item/toy/singlecard/une(src, src, "Red [i]", "[i]", REDCARD)
+		cards += new/obj/item/toy/singlecard/une(src, src, "Yellow [i]", "[i]", YELLOWCARD)
+		cards += new/obj/item/toy/singlecard/une(src, src, "Green [i]", "[i]", GREENCARD)
+		cards += new/obj/item/toy/singlecard/une(src, src, "Blue [i]", "[i]", BLUECARD)
+	for(var/i = 1; i <= 9; i++)//Second full set without a 0 card this time.
+		cards += new/obj/item/toy/singlecard/une(src, src, "Red [i]", "[i]", REDCARD)
+		cards += new/obj/item/toy/singlecard/une(src, src, "Yellow [i]", "[i]", YELLOWCARD)
+		cards += new/obj/item/toy/singlecard/une(src, src, "Green [i]", "[i]", GREENCARD)
+		cards += new/obj/item/toy/singlecard/une(src, src, "Blue [i]", "[i]", BLUECARD)
+	for(var/i = 0; i <= 3; i++)//Black is just a placeholder and not actually used in the coloring.
```
Comment (ShiftyRail) [inline code/modules/games/cards/une/une_cards.dm]: `var/i = 0 to 3` is prefered, I think

**Comment by DrSnips** [comment]
Open to suggestions on how to get a deck. Im thinking cargo merch computer.

**Comment by soyconsumer** [comment]
Mech computer sounds good, maybe leave a deck in the library

File: code/modules/games/cards/une/une_cards.dm (hunk 1 above)
Comment (DrSnips) [reply code/modules/games/cards/une/une_cards.dm]: I don't really see var/i=0 to X being used anywhere else. Got anywhere else that uses this?

Comment (DamianX) [reply code/modules/games/cards/une/une_cards.dm]: he's right in that it's marginally faster but it literally doesn't matter

**Comment by spoonylove** [comment]
A fun way to gamble. Nice. I think merch computer is a good place for it. Traders should also have a deck as well somewhere for them.

File: code/modules/games/cards/une/une_cards.dm (hunk 1 above)
Comment (Exxion) [reply code/modules/games/cards/une/une_cards.dm]: It's not an official standard, but it's preferred
>implying we have official standards

**Comment by DrSnips** [comment]
Everything should be good to go here. I tested it as much as I can but I require live testing to see how things work client side due to _planes_ and how people decide to play the game. I found throwing the cards on one tile works ok but some people might have issues.

**Pull request merged**